torch/testing/_internal/common_utils.py

```diff
@@ -2315,3 +2316,42 @@ def disable_gc():
             gc.enable()
     else:
         yield
+
+# Returns scalar tensor representation of a list of integer byte values
+def bytes_to_scalar(byte_list: List[int], dtype: torch.dtype, device: torch.device):
```

Comment:
Wondering if you need to unit test this. I guess this is being cross tested with Tensor storage so not a big deal

Reply:
Test added